documentation/modules/operators/proc-changing-topic-operator-mode.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-topic-operator.adoc
+
+[id='con-changing-topic-operator-mode-{context}']
+= (Preview) Switching between Topic Operator modes
```

Comment:
Should we remove "(Preview)" now that UTO is in beta?

Reply:
Looks like they've been removed!